base_tier_validation_definition_server_action/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+To define the server action,
+* By python code choose the **Server Action** option in the Definition field.
```

Comment:
Could you elaborate a bit more how to configure?

- What content/code is expected in the server action? (to return a variable to return a boolean?)
- Which model should you use to evaluate the server action? the same as the tier.definition? is possible to use a different one?

Reply:
@LoisRForgeFlow its updated.

Thanks.